.github/workflows/ci.yml

```diff
@@ -14,7 +14,7 @@ jobs:
 
         strategy:
             matrix:
-                node-version: [16.x, 18.x, 20.x]
+                node-version: [18.x, 20.x]
```

Comment:
suggestion: Let's not drop node 16 for now

Reply:
if you say so